server/README.md

```diff
@@ -82,111 +82,14 @@ The server executable can be built with `nix build .#cardano-trasaction-lib-serv
 "b8fe9f7f6255a6fa08f668ab632a8d081ad87983c77cd274e48ce450f0b349fd"
 ```
 
-## GET /fees
+## GET /eval-ex-units
```

Comment:
Looks like you accidentally removed some docs from README.

Reply:
I've actually removed the part of the docs that were duplicated

Reply:
Are you sure? It looks like a huge amount of the API docs are missing. How did you generate them?

Reply:
Oh, never mind. Sorry @errfrom I see what you mean now